annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -73,12 +75,7 @@
     private String enrichmentFields;
 
     private AnnotationControllerApi apiClient;
-
-    private MutationRecord mRecord;
-    private VariantAnnotation gnResponse;
```

Comment:
`mRecord`, `gnResponse`, and `canonicalTranscript` are no longer class variables, instead passed around as parameters to the methods.